### PR TITLE
Update interns 2023 blog post

### PR DESCRIPTION
**Pull request by LysandreJik**

### CODE REVIEW

**Comment by LysandreJik** [comment]
@thomasw21 @TevenLeScao do you want to add yours?

**Comment by LysandreJik** [comment]
@natolambert as well, regarding RLHF if you're ready to do so! (can be in a following PR if not!)

File: interns-2023.md [hunk 1]
```diff
@@ -52,9 +51,7 @@ The following Science team positions are available:
 * [Multimodal AI Internship](https://apply.workable.com/huggingface/j/C6B723A526/), working at the intersection of natural language processing and computer vision for the next generation of large multimodal models.
 * [OCR Internship](https://apply.workable.com/huggingface/j/CFDB7592D8/), incorporating textual information directly into multimodal models.
 * [Video Pretraining Internship](https://apply.workable.com/huggingface/j/16CC3BDDC9/), exploring pretraining for the video modality.
-* [Retrieval Augmentation Internship](https://apply.workable.com/huggingface/j/785DC30840/), investigating “semiparametric” retrieval augmentation methods for more consistent, more interpretable, and more efficient language models.
 * [Embodied AI Internship](https://apply.workable.com/huggingface/j/B3CDE6C150/), working with the Embodied AI team on reinforcement learning in simulators.
```
Comment (natolambert) [inline interns-2023.md:54]:
```suggestion
* [Embodied AI Internship](https://apply.workable.com/huggingface/j/B3CDE6C150/), working with the Embodied AI team on reinforcement learning in simulators. _Note_, update Jan. 2023: this internship is very likely to be focused on reinforcement learning from human feedback (RLHF). 
```

Comment (natolambert) [reply interns-2023.md:54]: @LysandreJik is that okay?

Comment (natolambert) [reply interns-2023.md:53]: Yeah, I think we are happy with the candidate pool for the option of 1 or both of these projects.

**Pull request merged**